history/cultures/furbolg_group.txt

```diff
@@ -14,6 +14,8 @@
 	discover_innovation = innovation_motte 
```

Comment:
I think it will be a continuity error to do so.  When special units are unlocked in vanilla the player doesn't automatically lose the ambiguous base unit.  The player most likely won't use the base unit, but the choice is still there at least since there are still both pros(costs) and cons(quality) to the base units.

Reply:
If I understood zumbak correctly, it is about the MaAs that are unlocked from the very start. Like, Night elves not having basic archers and sentinels, but only sentinels.

Reply:
> When special units are unlocked in vanilla the player doesn't automatically lose the ambiguous base unit.

The point is it's good for design. Players don't see options they're not supposed and efficient to pick. Vanilla units are another story for another PR. In @arithon's case, it will be nice too.

> Like, Night elves not having basic archers and sentinels, but only sentinels.

> If I understood zumbak correctly, it is about the MaAs that are unlocked from the very start.

Yes. Or like gryphons just replace aerial tech for humans, dwarves, etc.